src/__tests__/disableRanges-test.js

```diff
@@ -281,6 +281,29 @@ test("Nesting disabledRanges", t => {
   )
   planCount += 1
 
+  testDisableRanges(
+    `/* stylelint-disable */
+    /* stylelint-enable bar */
+    /* stylelint-enable foo */
+    /* stylelint-disable foo */
+    /* stylelint-enable bar */
+    /* stylelint-enable */`,
```

Comment:
@davidtheclark it is cover https://github.com/stylelint/stylelint/blob/master/src/disableRanges.js#L85